screens/settings/SettingsScreen.js

```diff
@@ -0,0 +1,130 @@
+import { FontAwesome5 } from '@expo/vector-icons';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  AsyncStorage,
+  Image,
+  ScrollView,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import {
+  Body,
+  NavButton,
+  NavHeaderContainer,
+  NavTitle,
+  Subhead,
+} from '../../components/BaseComponents';
+import CategoryBar from '../../components/resources/CategoryBar';
+import SettingsCard from '../../components/settings/SettingsCard';
+import Colors from '../../constants/Colors';
+import RecordIds from '../../constants/RecordIds';
+import { getCustomersById } from '../../lib/airtable/request';
+import { ResourceItemCard } from '../../styled/resources';
+
+export default class SettingsScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isGuest: false,
+      name: '',
+      number: '',
+    };
+  }
+
+  // Load customer record & transactions
+  async componentDidMount() {
+    const customerId = await AsyncStorage.getItem('customerId');
+    const isGuest = customerId === RecordIds.guestCustomerId;
+    if (customerId != null && !isGuest) {
+      const customer = await getCustomersById(customerId);
+      const { name } = customer;
+      const number = customer.phoneNumber;
+      this.setState({ name, number });
+    }
+    this.setState({ isGuest });
+  }
+
+  _logout = async () => {
+    await AsyncStorage.clear();
+    this.props.navigation.navigate('Auth');
+  };
+
+  render() {
+    return (
+      <View>
+        <NavHeaderContainer>
+          <NavButton onPress={() => this.props.navigation.goBack(null)}>
+            <FontAwesome5 name="arrow-left" solid size={24} />
+          </NavButton>
+          <NavTitle>Settings</NavTitle>
+        </NavHeaderContainer>
+        <ScrollView>
+          <CategoryBar icon="user" title="Account" />
+          {this.state.isGuest && (
+            <SettingsCard
+              title="Create an account"
+              description="Start earning healthy rewards."
+              path="Auth"
+              navigation={this.props.navigation}
```

Comment:
FYI this kind of `goBack` is also in `RewardsScreen` when people sign up from guest mode so they're taken back to `MapScreen` instead of seeing the outdated guest mode stuff in `RewardsScreen`